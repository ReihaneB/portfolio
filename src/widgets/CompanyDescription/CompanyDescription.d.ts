export interface CompanyDescriptionProps {
  description: string;
}
