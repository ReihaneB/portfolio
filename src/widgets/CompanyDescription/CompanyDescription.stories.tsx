import type { Meta, StoryObj } from '@storybook/react';

import CompanyDescription from './CompanyDescription';

const meta: Meta<typeof CompanyDescription> = {
  title: 'Widgets/CompanyDescription',
  component: CompanyDescription,
};

export default meta;
type Story = StoryObj<typeof CompanyDescription>;

export const Playground: Story = {
  args: {
    description: 'Dolore aliquip dolor consectetur esse eu non.',
  },
};

export const LongDescription: Story = {
  args: {
    description: 'Non reprehenderit irure ex occaecat duis nulla officia dolore magna Lorem duis. Dolore culpa aute aliquip sint eu irure aute mollit aliqua. Officia dolore et laboris proident esse deserunt irure minim magna ad ut sint. Incididunt elit Lorem qui cillum dolore duis labore laborum consequat minim. Veniam excepteur incididunt ad labore. Sint pariatur irure consequat id esse anim esse qui. Est adipisicing sit irure aliqua sint culpa nostrud ut laboris reprehenderit Lorem pariatur eiusmod cillum. Cillum magna mollit minim non proident dolor mollit aliquip eu voluptate duis aliquip voluptate. Officia reprehenderit officia amet anim. Magna officia quis aliqua officia anim pariatur officia sit esse laborum. Cupidatat cupidatat id incididunt officia et id cillum. Sint reprehenderit consectetur aute do enim do exercitation ex adipisicing ipsum proident magna consequat. Culpa dolore commodo ullamco occaecat aute cillum cupidatat ad ipsum sint non ea laboris magna.',
  },
};

export const ShortDescription: Story = {
  args: {
    description: 'Dolor qui amet Lorem aliquip elit sit anim. Adipisicing cupidatat cillum mollit sunt cupidatat cupidatat voluptate ex laborum aute irure. Dolore sit cillum anim eiusmod sit quis dolore id id cupidatat laborum culpa.',
  },
};
