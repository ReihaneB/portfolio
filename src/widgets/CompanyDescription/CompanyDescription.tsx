import type { CompanyDescriptionProps } from './CompanyDescription.d';
import styles from './CompanyDescription.module.css';

function CompanyDescription({
  description,
  ...rest
}: CompanyDescriptionProps) {
  return (
    <div {...rest}>
      <h2 className={styles.title}>Contexte</h2>
      <p>{description}</p>
    </div>
  );
}

export default CompanyDescription;
