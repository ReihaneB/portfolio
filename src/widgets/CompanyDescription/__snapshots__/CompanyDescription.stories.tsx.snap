// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Widgets/CompanyDescription LongDescription smoke-test 1`] = `
<main class="-storybook-preview-module__root--fQckk inter-normal">
  <div>
    <h2 class="src-widgets-CompanyDescription-CompanyDescription-module__title--hrKK8">
      Contexte
    </h2>
    <p>
      Non reprehenderit irure ex occaecat duis nulla officia dolore magna Lorem duis. Dolore culpa aute aliquip sint eu irure aute mollit aliqua. Officia dolore et laboris proident esse deserunt irure minim magna ad ut sint. Incididunt elit Lorem qui cillum dolore duis labore laborum consequat minim. Veniam excepteur incididunt ad labore. Sint pariatur irure consequat id esse anim esse qui. Est adipisicing sit irure aliqua sint culpa nostrud ut laboris reprehenderit Lorem pariatur eiusmod cillum. Cillum magna mollit minim non proident dolor mollit aliquip eu voluptate duis aliquip voluptate. Officia reprehenderit officia amet anim. Magna officia quis aliqua officia anim pariatur officia sit esse laborum. Cupidatat cupidatat id incididunt officia et id cillum. Sint reprehenderit consectetur aute do enim do exercitation ex adipisicing ipsum proident magna consequat. Culpa dolore commodo ullamco occaecat aute cillum cupidatat ad ipsum sint non ea laboris magna.
    </p>
  </div>
</main>
`;

exports[`Widgets/CompanyDescription Playground smoke-test 1`] = `
<main class="-storybook-preview-module__root--fQckk inter-normal">
  <div>
    <h2 class="src-widgets-CompanyDescription-CompanyDescription-module__title--hrKK8">
      Contexte
    </h2>
    <p>
      Dolore aliquip dolor consectetur esse eu non.
    </p>
  </div>
</main>
`;

exports[`Widgets/CompanyDescription ShortDescription smoke-test 1`] = `
<main class="-storybook-preview-module__root--fQckk inter-normal">
  <div>
    <h2 class="src-widgets-CompanyDescription-CompanyDescription-module__title--hrKK8">
      Contexte
    </h2>
    <p>
      Dolor qui amet Lorem aliquip elit sit anim. Adipisicing cupidatat cillum mollit sunt cupidatat cupidatat voluptate ex laborum aute irure. Dolore sit cillum anim eiusmod sit quis dolore id id cupidatat laborum culpa.
    </p>
  </div>
</main>
`;
